core-models/src/main/java/com/anytypeio/anytype/core_models/Command.kt

```diff
@@ -545,4 +545,29 @@ sealed class Command {
     data class ProcessCancel(
         val processId: Id
     ) : Command()
+
+    data class GetVersions(
```

Comment:
could we wrap all these new commands into `VersionHistory` class?

Reply:
yes

Reply:
done